justsendmail/mail.py

```diff
@@ -60,15 +62,29 @@ def send_mail(
     msg.attach(body)
 
     if attachments:
-        for k, v in attachments.items():
-            part = MIMEBase("application", "octet-stream")
-            part.set_payload(open(v, "rb").read())
-            encoders.encode_base64(part)
-            part.add_header(
-                "Content-Disposition",
-                'attachment; filename="{}"'.format(os.path.basename(k)),
-            )
-            msg.attach(part)
+        if isinstance(attachments, dict):
+            for k, v in attachments.items():
+                part = MIMEBase("application", "octet-stream")
+                part.set_payload(open(v, "rb").read())
+                encoders.encode_base64(part)
+                part.add_header(
+                    "Content-Disposition",
+                    'attachment; filename="{}"'.format(os.path.basename(k)),
+                )
+                msg.attach(part)
+        else:
+            # Assume it's a general iterable
+            for attachment in attachments:
+                part = MIMEBase("application", "octet-stream")
```

Comment:
DRY

Reply:
Done in 704e60a087ee476d3c04288bfc0b800766c85a6e

Will test it tomorrow